go/pkg/fakes/exec.go

```diff
@@ -144,7 +145,7 @@ func (s *Exec) Execute(req *repb.ExecuteRequest, stream regrpc.Execution_Execute
 	} else if err = stream.Send(op); err != nil {
 		return err
 	}
-	s.numExecCalls++
+	atomic.AddInt32(&s.numExecCalls, 1)
```

Comment:
Also wrap the call in Clear()? Sorry github won't let me comment on an unchanged line